File: RECORRIDO INVERSO/recorridoinverso.cpp
#include <iostream>
using namespace std;

int main() {
    int ar[] = {30, 40, 50, 60, 70, 80};
    int n = sizeof(ar) / sizeof(ar[0]);

    cout << "El recorrido inverso del array es: ";
    for (int d = n - 1; d >= 0; d--) {
        cout << ar[d] << " ";
    }
    cout << endl;

    return 0;
}

File: RECORRIDO INVERSO/recorridoinverso.cs
using System;
class Program
{
    static void Main()
    {
        int[] arr= {30, 40, 50, 60, 70, 80};
        int size = arr.Length;
        Console.WriteLine("El recorrido inverso del array es: ");
        for (int g= size - 1; g >= 0; g--)
        {
            Console.Write(arr[g]+ " ");
        }
    }
 }  